src/xcode/ENA/ENA/Source/Services/__tests__/Mocks/MockTestStore.swift

```diff
@@ -8,12 +8,23 @@ import OpenCombine
 #if !RELEASE
 
 final class MockTestStore: Store, PPAnalyticsData {
+
+	init() {
+#if DEBUG
+		if isUITesting {
+			self.showAnotherHighExposureAlert = LaunchArguments.risk.anotherHightEncounter.boolValue
+		}
+#endif
```

Comment:
For `userNeedsToBeInformedAboutHowRiskDetectionWorks` we do this inside the `HomeTableViewController`. But not sure which version I prefer, only think maybe we can consolidate this, maybe even makes sense to move it to the store as well 🤔

Reply:
I think it makes more sense to move it into the mock, this will keep the production code cleaner